$NetBSD: patch-man_CMakeLists.txt,v 1.1 2025/03/10 22:54:38 gdt Exp $

Ask cmake config to respect CMAKE_INSTALL_MANDIR.

\todo File upstream.

--- man/CMakeLists.txt.orig	2025-01-06 18:19:00.158960664 +0000
+++ man/CMakeLists.txt
@@ -62,9 +62,9 @@ if(MANPAGE_TARGETS)
 endif()
 
 if(ENABLE_INSTALL)
-    install(FILES osm2pgsql.1 DESTINATION share/man/man1)
-    install(FILES osm2pgsql-replication.1 DESTINATION share/man/man1)
+    install(FILES osm2pgsql.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
+    install(FILES osm2pgsql-replication.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
     if (BUILD_GEN)
-        install(FILES osm2pgsql-gen.1 COMPONENT gen EXCLUDE_FROM_ALL DESTINATION share/man/man1)
+        install(FILES osm2pgsql-gen.1 COMPONENT gen EXCLUDE_FROM_ALL DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
     endif()
 endif()
